### DISCUSSION TITLE
Passing config proxy to Cygwin install

### DISCUSSION THREAD

**Discussion by drewbitt** [ScoopInstaller/Main#4330] [Q&A] [answered]
Scoop setup with the proxy config "scoop config proxy" will download Cygwin with that proxy, but in order to complete the Cygwin install Cygwin will download from the mirror in the scoop file for additional packages. It will not use the scoop config proxy for the additional downloads, and the scoop install will fail if it cannot install the setup packages. 

Is there a way for me to send a package additional args in scoop?

**Comment by rashil2000** [comment, marked as answer]
In the pre_install script, you can check for the existence of Scoop proxy with the simple command `scoop config proxy`, and then proceed accordingly
